card/db/migrate_core_cards/data/responsive_sidebar.json

```diff
@@ -0,0 +1,26 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-05-01 03:06:37 +0200",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: right sidebar]]\n[[Default Layout]]\n[[*header]]",
+    "value": [
+      {
+        "name": "style: right sidebar",
+        "type": "SCSS",
+        "content": "/*\r\nThis stylesheet defines a simple layout with the sidebar at right.\r\n*/\r\n#primary {\r\n  margin: 0 1% 3em 1%;\r\n  float: left;\r\n  width: 67%;\r\n}\r\n\r\n#secondary {\r\n  margin: 0 1% 3em 1%;\r\n  float: right;\r\n  width: 28%;\r\n}\r\n\r\nfooter {\r\n  clear:both;\r\n  text-align: center;\r\n  padding: 1em;\r\n}\r\n\r\n@media print {\r\n  article,\r\n  #primary {\r\n    width: 100%;\r\n    float: none;\r\n    margin: 0;\r\n  }\r\n\r\n  aside,\r\n  #secondary,\r\n  footer {\r\n    display: none;\r\n  }\r\n}\r\n"
+      },
+      {
+        "name": "Default Layout",
+        "type": "Layout",
+        "content": "<!DOCTYPE HTML>\r\n<html>\r\n  <head>{{*head|core}}</head>\r\n  <body>\r\n    <header>{{*header|core}}\r\n    <div class=\"container\">\r\n      <div class=\"row row-offcanvas\">\r\n        <article class=\"col-xs-12 col-sm-9\">{{_main|open}}</article>\r\n        <aside class=\"col-xs-6 col-sm-3 offcanvas-right\">{{*sidebar|core}}</aside>\r\n      </div>\r\n      <footer>{{*footer|core}}</footer>\r\n    </div>\r\n  </body>\r\n</html>"
```

Comment:
We have to change the default layout again for this. I guess we don't want to do that with the next point release.